src/access_nri_intake/data/utils.py

```diff
@@ -48,21 +48,41 @@ def available_versions(pretty: bool = True):
     # Work out where the catalogs are stored
     base_path = _get_catalog_rp()
```

Comment:
Not immediately related to the changes on this PR but I only just realised we have a function called `get_catalog_fp` and another called `_get_catalog_rp` - I think we could make these more obviously distinct?

Reply:
Spent about 10 minutes reading this file before I even noticed they were two separate functions

Reply:
Yeah, that's fair. Just for clarity:

- `get_catalog_fp` returns the full file path to the `catalog.yaml` file;
- `_get_catalog_rp` reads the `catalog.yaml`, and computes the root path (`rp`) to the catalog data by stripping off the `{{version}}/metacatalog.yaml` from the data path therein.

Reply:
Makes sense. Maybe `_get_catalog_rp` => `_get_catalog_root` given it's a local function & so easier to track down the definition/meaning?